#-----------------------------------------------------------
# Vivado v2019.1 (64-bit)
# SW Build 2552052 on Fri May 24 14:49:42 MDT 2019
# IP Build 2548770 on Fri May 24 18:01:18 MDT 2019
# Start of session at: Wed May  4 17:28:31 2022
# Process ID: 22388
# Current directory: D:/CodeTry/CODExperiment/Lab5/project_1
# Command line: vivado.exe -gui_launcher_event rodinguilauncherevent97572 D:\CodeTry\CODExperiment\Lab5\project_1\project_1.xpr
# Log file: D:/CodeTry/CODExperiment/Lab5/project_1/vivado.log
# Journal file: D:/CodeTry/CODExperiment/Lab5/project_1\vivado.jou
#-----------------------------------------------------------
start_gui
open_project D:/CodeTry/CODExperiment/Lab5/project_1/project_1.xpr
Scanning sources...
Finished scanning sources
INFO: [IP_Flow 19-234] Refreshing IP repositories
INFO: [IP_Flow 19-1704] No user IP repositories specified
INFO: [IP_Flow 19-2313] Loaded Vivado IP repository 'D:/Xilinx/Vivado/2019.1/data/ip'.
launch_runs impl_1 -to_step write_bitstream -jobs 6
[Wed May  4 17:28:51 2022] Launched synth_1...
Run output will be captured here: D:/CodeTry/CODExperiment/Lab5/project_1/project_1.runs/synth_1/runme.log
[Wed May  4 17:28:51 2022] Launched impl_1...
Run output will be captured here: D:/CodeTry/CODExperiment/Lab5/project_1/project_1.runs/impl_1/runme.log
open_hw
connect_hw_server
INFO: [Labtools 27-2285] Connecting to hw_server url TCP:localhost:3121
INFO: [Labtools 27-2222] Launching hw_server...
INFO: [Labtools 27-2221] Launch Output:

****** Xilinx hw_server v2019.1
  **** Build date : May 24 2019 at 15:13:31
    ** Copyright 1986-2019 Xilinx, Inc. All Rights Reserved.
ERROR: [Labtools 27-2220] Launch Error:
Unable to launch local hw_server executable.

ERROR: [Common 17-39] 'connect_hw_server' failed due to earlier errors.
reset_run impl_1
reset_run synth_1
exit
INFO: [Common 17-206] Exiting Vivado at Wed May  4 17:29:12 2022...
